8_visualize_metrics.yml

```diff
@@ -0,0 +1,220 @@
+                metric_file = '4_evaluation/out/exceedance_metrics_by_seg.csv',
+                metric_col = I('false_pos'),
+                plot_model = I('rgcn2_full_full'),
+                legend_text = I('RGNC-Model Exceedance (F-Pos)'))
+
+  # exceedance for sntemp model
+  8_visualize/out/sntemp_full_false_pos_on_network.png:
+    command: plot_metric_map(out_file = target_name,
+                network_ind = '1_network/out/network.rds.ind',
+                metric_file = '4_evaluation/out/exceedance_metrics_by_seg.csv',
+                metric_col = I('false_pos'),
+                plot_model = I('sntemp_full'),
+                legend_text = I('SNTEMP-Model Exceedance (F-Pos)'))
+
+  # exceedance for rgnc model
+  8_visualize/out/rgnc2_full_false_neg_on_network.png:
+    command: plot_metric_map(out_file = target_name,
+                network_ind = '1_network/out/network.rds.ind',
+                metric_file = '4_evaluation/out/exceedance_metrics_by_seg.csv',
+                metric_col = I('false_neg'),
+                plot_model = I('rgcn2_full_full'),
+                legend_text = I('RGNC-Model Exceedance (F-Neg)'))
+
+  # exceedance for sntemp model
+  8_visualize/out/sntemp_full_false_neg_on_network.png:
+    command: plot_metric_map(out_file = target_name,
+                network_ind = '1_network/out/network.rds.ind',
+                metric_file = '4_evaluation/out/exceedance_metrics_by_seg.csv',
+                metric_col = I('false_neg'),
+                plot_model = I('sntemp_full'),
+                legend_text = I('SNTEMP-Model Exceedance (F-Neg)'))
+
+  # max temps for rgnc model
+  8_visualize/out/rgnc2_full_max_temp_error_on_network.png:
+    command: plot_metric_map(out_file = target_name,
+                network_ind = '1_network/out/network.rds.ind',
+                metric_file = '4_evaluation/out/max_metrics_by_seg_year.csv',
+                metric_col = I('error_obs_pred'),
+                plot_model = I('rgcn2_full_full'),
+                legend_text = I('RGNC-Model Max Temps Error'))
+
+  # max temps for sntemp model
+  8_visualize/out/sntemp_full_max_temp_error_on_network.png:
+    command: plot_metric_map(out_file = target_name,
+                network_ind = '1_network/out/network.rds.ind',
+                metric_file = '4_evaluation/out/max_metrics_by_seg_year.csv',
+                metric_col = I('error_obs_pred'),
+                plot_model = I('sntemp_full'),
+                legend_text = I('SNTEMP-Model Max Temps Error'))
+
+# max temps for rgnc model
+  8_visualize/out/rgnc2_full_max_temp_time_on_network.png:
+    command: plot_metric_map(out_file = target_name,
+                network_ind = '1_network/out/network.rds.ind',
+                metric_file = '4_evaluation/out/max_metrics_by_seg_year.csv',
+                metric_col = I('error_max_timing'),
+                plot_model = I('rgcn2_full_full'),
+                legend_text = I('RGNC-Model Max Temps Time Error'))
```

Comment:
legend suggestion: "Max temp timing error (days)". Also, because these legends are long, you might want to wrap the text to two lines. This can be accomplished by adding a `\n` wherever you want the line to break. e.g., "Max temp timing\n error (days)"